tests/Unit/Domain/CoordinateMaps/Test_EquatorialCompression.cpp

```diff
@@ -46,36 +47,45 @@ void test_radius() {
   const double aspect_ratio = aspect_ratio_dis(gen);
   CAPTURE(aspect_ratio);
   const std::array<double, 3> input_point{{x, y, z}};
-  const CoordinateMaps::EquatorialCompression angular_compression_map(
-      aspect_ratio);
-  const auto result_point = angular_compression_map(input_point);
-  const auto& result_x = result_point[0];
-  const auto& result_y = result_point[1];
-  const auto& result_z = result_point[2];
 
-  // The EquatorialCompression map should preserve radial lengths.
-  CHECK(magnitude(result_point) == approx(magnitude(input_point)));
+  for (const auto& index_polar_axis : {0_st, 1_st, 2_st}) {
+    const CoordinateMaps::EquatorialCompression angular_compression_map(
+        aspect_ratio, index_polar_axis);
+    const auto result_point = angular_compression_map(input_point);
+
+    // The EquatorialCompression map should preserve radial lengths.
+    CHECK(magnitude(result_point) == approx(magnitude(input_point)));
+
+    // It is easy to mix up \alpha with one over \alpha,
+    // so this comment serves as a reminder to future readers
+    // as to which direction the \alpha should go in the
+    // CHECK statement.
+    //
+    // The aspect ratio is the ratio of the distance perpendicular to
+    // the polar axis to the distance along the polar axis for a given point.
+    // The parameter name `aspect_ratio` is used because points with
+    // tan \theta = 1, which have an aspect ratio of one, get mapped to points
+    // with tan \theta = \alpha, which have an aspect ratio of `aspect_ratio`.
+    //
+    // The angular compression map is:
+    //  tan \theta' = \alpha tan\theta.
+    // Then, if the argument provided to the EquatorialCompression
+    // map is, for example, 4, points on the undistorted sphere
+    // that have an angle tan \theta = sqrt(x^2 + y^2)/z are
+    // mapped to points that have an angle
+    //  tan \theta' = 4*sqrt(x^2+y^2)/z
+    //              = sqrt(x'^2 + y'^2)/z'.
+
+    const auto& planar_distance =
+        sqrt(pow<2>(input_point[(index_polar_axis + 1) % 3]) +
+             pow<2>(input_point[(index_polar_axis + 2) % 3]));
```

Comment:
Clang-tidy still isn't liking this indexing. It suggests `gsl::at` as well.  Here and below too, lines 80-87.

Same comment above about `square` vs `pow<2>`.